examples/apollo-service-example/src/main/java/com/spotify/zoltar/examples/apollo/IrisPredictor.java

```diff
@@ -79,9 +78,7 @@
     };
 
     final PredictorBuilder<TensorFlowModel, Iris, Example, Long> predictorBuilder =
-        Predictors
-            .newBuilder(modelLoader, extractFn, predictFn)
-            .with(Instrumentations.predictor(metrics));
+        Predictors.newBuilderWithMetrics(modelLoader, extractFn, predictFn, metrics);
```

Comment:
Actually wondering if it would make sense to force folks to use metrics and make this builder the primary one (and rm `WithMetrics` from the name)